package.json

```diff
@@ -40,6 +41,7 @@
     "i18n-nationality": "1.1.1",
     "lodash": "^4.17.20",
     "middy-middleware-json-error-handler": "^1.0.0",
+    "qrcode": "^1.4.4",
```

Comment:
Could the generation of the qr code be moved to that separate package? It should be able to compute it because it is given the url.

This would reduce the number of dependencies that this repo has too

Reply:
Sure, will move this over to `sns-notify-recipients`